### PR TITLE
Bug fix ROC report + some code cleanup

### PR DESCRIPTION
**Pull request by Nadav-Barak**
Resolves https://github.com/deepchecks/deepchecks/issues/2151

### CODE REVIEW

File: deepchecks/tabular/checks/model_evaluation/roc_report.py [hunk 1]
```diff
@@ -45,7 +46,7 @@ def __init__(self, excluded_classes: List = None,
                  random_state: int = 42,
                  **kwargs):
         super().__init__(**kwargs)
-        self.excluded_classes = excluded_classes or []
+        self.excluded_classes = excluded_classes
```
Comment (matanper) [inline deepchecks/tabular/checks/model_evaluation/roc_report.py:49]: why

**Review by matanper — APPROVED** [review]

**Review by noamzbr — APPROVED** [review]

File: deepchecks/tabular/checks/model_evaluation/roc_report.py (hunk 1 above)
Comment (Nadav-Barak) [reply deepchecks/tabular/checks/model_evaluation/roc_report.py:49]: so you could explicitly define to not ignore anything (currently in binary by default we ignore class 0)

Comment (Nadav-Barak) [reply deepchecks/tabular/checks/model_evaluation/roc_report.py:49]: will add it also in docstring

File: tests/tabular/checks/model_evaluation/roc_report_test.py [hunk 2]
```diff
@@ -50,7 +50,7 @@ def test_binary_classification(iris_binary_string_split_dataset_and_model):
     # Arrange
     train, _, clf = iris_binary_string_split_dataset_and_model
     # Act
-    result = RocReport().run(train, clf)
+    result = RocReport(excluded_classes=[]).run(train, clf)
```
Comment (matanper) [inline tests/tabular/checks/model_evaluation/roc_report_test.py:53]: why

**Pull request merged**